cmd/runtimetest/main.go

```diff
@@ -698,6 +698,14 @@ func validateMounts(spec *rspec.Spec) error {
 	return mountErrs
 }
 
+func validatePlatform(platform string) error {
+	if platform != "linux" && platform != "solaris" && platform != "windows" {
+		return fmt.Errorf("runtime-tools has not implemented testing for your platform %q, because the spec has nothing to say about it", platform)
```

Comment:
This is hard to say whether right or not.
Though spec don't have specific configs for those platform, But spec has common configs for all platform.
And spec seems never say it just supports those three platforms.